e2e/wdio/README.md

```diff
@@ -0,0 +1,18 @@
+# Introduction
+WDIO Automation
+
+# Getting Started
+
+Prereq - app to be tested must already be deployed and has access to experience editor
+
+1.	Run `$npm i`
+2.	Update .env fields BASEURL, USER_NAME, PASSWORD, and APPNAME or can provide values from cmdline
+3.  To run tests: 
+    - Run single spec file - `set "APPNAME=JssNextWeb" & set "PASSWORD=b" && npx wdio wdio.conf.js --baseUrl https://ssge2ecm --spec ./test/specs/experience-editor.spec.js`
+    - Run suite as defined in wdio.conf.js - `set "APPNAME=etoenextjs" & set "PASSWORD=b" && npx wdio wdio.conf.js --baseUrl https://cm.jssdev.localhost --suite ee`
+4. To run setup script for new fresh environments `set "APPNAME=etoenextjs" & set "PASSWORD=b" && npx wdio wdio.conf.js --baseUrl https://cm.jssdev.localhost --suite setup`. This will: 
+    - enable admin permission for JssImport user in user manage page if not already enabled
+    - populate solr schema and indexing manager in control panel page
+    - publish content items and resolve suggested fix in workbox page
```

Comment:
Some of these items can be done with the Sitecore CLI. I see there's usage in the `install_sitecore_cli.ps1` file already. Should we being doing what we can with CLI first, JS as next-best-option?